ee/server/service/devices.go

```diff
@@ -16,6 +18,43 @@ func (svc *Service) RequestEncryptionKeyRotation(ctx context.Context, hostID uin
 	return svc.ds.SetDiskEncryptionResetStatus(ctx, hostID, true)
 }
 
+type migrateMDMDevicePayload struct {
+	Timestamp time.Time `json:"timestamp"`
+	Host      struct {
+		ID             uint   `json:"id"`
+		UUID           string `json:"uuid"`
+		HardwareSerial string `json:"hardware_serial"`
+	} `json:"host"`
+}
+
+func (svc *Service) TriggerMigrateMDMDevice(ctx context.Context, host *fleet.Host) error {
```

Comment:
I considered adding this to the `webhooks` package, but wasn't sure the best way encapsulate it as both a webhook and a premium-only feature.

Reply:
I'm not sure but I think the license/`ee` requirement is more important in this case.